doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -164,12 +164,18 @@ nRF Desktop
   * Support for the nRF54L15 PDK with the ``nrf54l15pdk_nrf54l15_cpuapp`` board target.
     The PDK can act as a sample mouse or keyboard.
     It supports the Bluetooth LE HID data transport and uses SoftDevice Link Layer with Low Latency Packet Mode (LLPM) enabled.
+    The PDK uses MCUboot bootloader built in the direct-xip mode (``MCUBOOT+XIP``), and supports the firmware updates using the :ref:`nrf_desktop_dfu`.
   * The ``nrfdesktop-wheel-qdec`` DT alias support to :ref:`nrf_desktop_wheel`.
     You must use the alias to specify the QDEC instance used for scroll wheel, if your board supports multiple QDEC instances (for example ``nrf54l15pdk_nrf54l15_cpuapp``).
     You do not need to define the alias if your board supports only one QDEC instance, because in that case, the wheel module can rely on the ``qdec`` DT label provided by the board.
 
-* Updated the :kconfig:option:`CONFIG_BT_ADV_PROV_TX_POWER_CORRECTION_VAL` Kconfig option value in the nRF52840 Gaming Mouse configurations with the Fast Pair support.
-  The value is now aligned with the Fast Pair requirements.
+* Updated:
+
+  * The :kconfig:option:`CONFIG_BT_ADV_PROV_TX_POWER_CORRECTION_VAL` Kconfig option value in the nRF52840 Gaming Mouse configurations with the Fast Pair support.
+    The value is now aligned with the Fast Pair requirements.
+  * Enabled configuration channel HID output report (:ref:`CONFIG_DESKTOP_CONFIG_CHANNEL_OUT_REPORT <config_desktop_app_options>`) for nRF Desktop peripherals with :ref:`nrf_desktop_dfu`.
+    This mitigates HID report rate drops during DFU image transfer through nRF Desktop dongle.
+    The output report was also enabled for ``nrf52bkd_nrf52832`` in debug configuration for consistency with release configuration.
```

Comment:
```suggestion
  * Enabled the :ref:`CONFIG_DESKTOP_CONFIG_CHANNEL_OUT_REPORT <config_desktop_app_options>` Kconfig option for the nRF Desktop peripherals with :ref:`nrf_desktop_dfu`.
    The option mitigates HID report rate drops during DFU image transfer through the nRF Desktop dongle.
    The output report is also enabled for the ``nrf52kbd_nrf52832`` build target in the debug configuration to maintain consistency with the release configuration.
```
Tried to make it flow better, let me know in case the meaning is not as intended